src/@layout.latte

```diff
@@ -106,7 +106,7 @@
 
 <div id="right">
 <div id="rightInner">
-	<form{if $config->googleCseId} action="http://www.google.com/cse"{/if} id="search">
+	<form{if $config->googleCseId} action="https://www.google.com/cse"{/if} id="search">
```

Comment:
Maybe better to completely skip the scheme and let the client auto-prefix that? Something like this should do:

```
action="//www.google.com/cse"
```
```

Thoughts?